src/main/java/com/github/fairit/be_doyou_destinypro/controller/PlayerController.java

```diff
@@ -1,25 +1,30 @@
 package com.github.fairit.be_doyou_destinypro.controller;
 
-import com.github.fairit.be_doyou_destinypro.dto.player.Player;
-import com.github.fairit.be_doyou_destinypro.dto.player.api.PlayerApi;
+import com.github.fairit.be_doyou_destinypro.service.character_service.CharacterApiService;
 import com.github.fairit.be_doyou_destinypro.service.player_service.PlayerService;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.List;
+
+import static org.springframework.web.bind.annotation.RequestMethod.POST;
+
 @RestController
 public class PlayerController {
 
     private PlayerService playerService;
+    private CharacterApiService characterApiService;
 
     @Autowired
-    public PlayerController(final PlayerService playerService) {
+    public PlayerController(final PlayerService playerService, final CharacterApiService characterApiService) {
         this.playerService = playerService;
+        this.characterApiService = characterApiService;
     }
 
-    @PostMapping("/find")
-    public PlayerApi searchPlayerByNickname(@RequestBody Player player) {
-        return playerService.findByPlayerNickname(player);
+    @RequestMapping(value = "/findplayer/{nickname}", method = POST)
```

Comment:
I don't sure it's good way to do it. `@PostMapping` `@RequestMapping` or something else?

Reply:
@RequestMapping is OK